storage/interface.go

```diff
@@ -158,13 +158,13 @@ func (f QueryableFunc) Querier(ctx context.Context, mint, maxt int64) (Querier,
 // Operations on the Appender interface are not goroutine-safe.
 type Appender interface {
 	// Append adds a sample pair for the given series.
-	// An optional reference number can be provided to accelerate calls.
-	// A reference number is returned which can be used to add further
-	// samples in the same or later transactions.
```

Comment:
> the same or later transactions

I assumed this means `commit` transactions?

Reply:
Not sure I quite follow - commit implies the end of a transaction, but I don't think I've seen them called `commit` transactions before.

Reply:
Right, what I mean is the following: Can we use refs across transactions?